src/RestApi/StoreApi/Schemas/CartSchema.php

```diff
@@ -176,12 +185,13 @@ public function get_item_response( $cart ) {
 		$context            = 'edit';
 
 		return [
-			'coupons'        => array_values( array_map( [ $cart_coupon_schema, 'get_item_response' ], array_filter( $cart->get_applied_coupons() ) ) ),
-			'items'          => array_values( array_map( [ $cart_item_schema, 'get_item_response' ], array_filter( $cart->get_cart() ) ) ),
-			'items_count'    => $cart->get_cart_contents_count(),
-			'items_weight'   => wc_get_weight( $cart->get_cart_contents_weight(), 'g' ),
-			'needs_shipping' => $cart->needs_shipping(),
-			'totals'         => (object) array_merge(
+			'coupons'           => array_values( array_map( [ $cart_coupon_schema, 'get_item_response' ], array_filter( $cart->get_applied_coupons() ) ) ),
+			'selected_shipping' => WC()->session->get( 'chosen_shipping_methods' ),
```

Comment:
Do we need to normalise this to match schema? Anything can set session data, so it might be good to ensure it matches the expected format before returning it.

Reply:
This makes sense, I think we can return an empty array here, will write the logic to another method.